database/schema.js

```diff
@@ -1,75 +1,45 @@
 var mongoose = require('mongoose');
 var Promise = require('bluebird');
 var mpUtils = require('mongoose-bluebird-utils');
-mongoose.connect('mongodb://localhost/musetrap'); //will probably change
+mongoose.connect('mongodb://localhost/musetrap'); 
 var db = mongoose.connection;
 db.on('error', console.error);
 
 var Schema = mongoose.Schema;
 
 var sequenceSchema = new Schema ({
   id: {type:Schema.Types.ObjectId, unique: true},
-  userID: Number,
-  name: {type: String, unique: true, required: true},
-  sequenceRows: [[]],  // Mongo does not have an 'object' type, so need to use mixed. has some repercussions to discuss
-  bpm: Number,
-  beats: [],
+  user: String,
+  sequenceRows: [{}],
   shareable: Boolean
 })
 
-/** This is the schema for the Users collection in Mongo */
+
 var userSchema = new Schema({
-  id: {type:Schema.Types.ObjectId, unique: true},
-  sequences: [sequenceSchema], //according to mongo docs, this is how you define an array of a different schema
+  id: {type:Schema.Types.ObjectId, unique: true}, 
   userName: {type: String, unique: true, required: true},
   passWord: String
 })
-/** This is the schema for the Sequences collection in Mongo */
 
-/** This is the schema for the Samples collection in Mongo */
 var sampleSchema = new Schema ({
   id: Schema.Types.ObjectId,
-  location: String, //will be a url
-  name: String,
-  category: String
+  url: String, 
+  name: String
 })
 
+var soundBoardMatrix = [
+  {beat: undefined, row: [0, 0, 0, 0, 0, 0, 0, 0]},
+  {beat: undefined, row: [0, 0, 0, 0, 0, 0, 0, 0]},
+  {beat: undefined, row: [0, 0, 0, 0, 0, 0, 0, 0]},
+  {beat: undefined, row: [0, 0, 0, 0, 0, 0, 0, 0]}
+]
 
-var soundBoardMatrix = {
-  beats: [undefined, undefined, undefined, undefined],
-  bpm: 120,
-  sequenceRows: [
-    [0,0,0,0,0,0,0,0],
-    [0,0,0,0,0,0,0,0],
-    [0,0,0,0,0,0,0,0],
-    [0,0,0,0,0,0,0,0]
-    ],
-}
-
-
-let newUser = function(name, password) { //function to create a new user- probably will be needed at login page
+let newUser = function(name, password) { 
  var user = new Users (
-    {sequences: [],
-    userName: name,
+    { userName: name,
     passWord: password
   });
   return user.saveAsync();
-
-let updateUser = function(sequence) {
-  var promise = Users.findById(sequence.userID).exec();
-
-  promise.then(function(user) {
-    user.sequences.push(sequence);
-
-    return user.save(); 
-  })
-  .then(function(user) {
-    console.log('updated user: ' + user.name);
-  })
-  .catch(function(err){
-    console.log('error:', err);
-  });
-}
 }
 
 let updateSequence = function(sequence) {
```

Comment:
I'm going to leverage updateSequence to also make the song shareable...
could you add "newSequence.shareable = sequence.shareable;" before newSequence.sequenceRows = sequence.sequenceRows; ?
    newSequence.shareable = sequence.shareable;
    newSequence.sequenceRows = sequence.sequenceRows;